docs/compilers/CSharp/Compiler Breaking Changes - DotNet 5.md

```diff
@@ -58,3 +58,11 @@
             o is sbyte or short or int or long;
     ```
     Because the `and` and `or` combinators can follow a type pattern, the compiler interprets them as part of the pattern combinator rather than an identifier for the declaration pattern. Consequently, it is an error to use `or` or `and` as pattern variable identifiers starting with C# 9.0.
+
+4. The fix for https://github.com/dotnet/roslyn/issues/44067 generates correct (different) code.
```

Comment:
>generates correct (different) code. [](start = 61, length = 35)

Did we get compatibility council approval for this behavior change in the compiler? If not, I don't think it is blocking this PR, but we should follow up.

Reply:
The compat council has the issue on their plate.

Reply:
> The compat council has the issue on their plate.

Please send me an invite for the discussion. 

---
In reply to: [442950442](https://github.com/dotnet/roslyn/pull/44025#discussion_r442950442) [](ancestors = 442950442)

Reply:
This is being tracked by [https://github.com/dotnet/roslyn/issues/45453](https://github.com/dotnet/roslyn/issues/45453)

---
In reply to: [443144872](https://github.com/dotnet/roslyn/pull/44025#discussion_r443144872) [](ancestors = 443144872,442950442)